src/uct/ib/rc/base/rc_iface.c

```diff
@@ -678,6 +692,177 @@ unsigned uct_rc_iface_do_progress(uct_iface_h tl_iface)
     return iface->progress(iface);
 }
 
+#if HAVE_IBV_EXP_DM
+const uint32_t uct_rc_mlx5_dm_key = 1245;
```

Comment:
Pls explain why 1245

Reply:
great random, it was 12:45 pm when I added this code

Reply:
Pls add the comment